src/tpg_test_mgmt_cli.c

```diff
@@ -186,6 +186,78 @@ cmdline_parse_inst_t cmd_tests_stop = {
         NULL,
     },
 };
+/****************************************************************************
+ * - "show link rate"
+ ****************************************************************************/
+struct cmd_show_link_rate_result {
+    cmdline_fixed_string_t show;
+    cmdline_fixed_string_t link;
+    cmdline_fixed_string_t rate;
+};
+
+static cmdline_parse_token_string_t cmd_show_link_rate_T_show =
+TOKEN_STRING_INITIALIZER(struct cmd_show_link_rate_result, show, "show");
+static cmdline_parse_token_string_t cmd_show_link_rate_T_link =
+TOKEN_STRING_INITIALIZER(struct cmd_show_link_rate_result, link, "link");
+static cmdline_parse_token_string_t cmd_show_link_rate_T_rate =
+TOKEN_STRING_INITIALIZER(struct cmd_show_link_rate_result, rate, "rate");
+
+static void tpg_print_line_rate(void)
```

Comment:
With proposed prototype of test_show_link_rate, How can we have two different kinds of printing formats in test_show_link_rate for CLI(tpg_printf) and UI(UI_PRINTLN_WIN) which accepts different arguments to have generic way of printing?